src/swap/defi/bridgeless.ts

```diff
@@ -353,7 +356,50 @@ export function makeBridgelessPlugin(
 
       const newRequest = await getMaxSwappable(fetchSwapQuoteInner, request)
       const swapOrder = await fetchSwapQuoteInner(newRequest)
-      return await makeSwapPluginQuote(swapOrder)
+
+      const swapPluginQuote = await makeSwapPluginQuote(swapOrder)
+
+      const out = {
+        ...swapPluginQuote,
+        async approve(_opts?: EdgeSwapApproveOptions): Promise<EdgeSwapResult> {
+          const swapResult = await swapPluginQuote.approve(_opts)
+          const { txid } = swapResult.transaction
+
+          if (
+            swapResult.transaction.savedAction?.actionType === 'swap' &&
+            swapResult.transaction.savedAction.orderId != null
+          ) {
+            swapResult.transaction.savedAction.orderId.replace('{{TXID}}', txid)
+
+            const [
+              chainId,
+              ,
+              txNonce
+            ] = swapResult.transaction.savedAction.orderId.split('/')
+
+            opts.io
```

Comment:
await this. if it fails throw an error as the user should know that the swap will likely not complete. An error like

'Swap transaction sent but error creating withdrawal transaction. Contact [support email] if you do not receive funds within 2 hours'

Reply:
I disagree. This will leave the user on the confirmation scene where they could slide to confirm again. It would be possible to burn the Zano assets multiple times.